packages/runner/src/studio/studio-recorder.js

```diff
@@ -314,81 +316,30 @@ export class StudioRecorder {
     return type
   }
 
-  _addModifierKeys = (key, event) => {
-    const { altKey, ctrlKey, metaKey, shiftKey } = event
-
-    return `{${altKey ? 'alt+' : ''}${ctrlKey ? 'ctrl+' : ''}${metaKey ? 'meta+' : ''}${shiftKey ? 'shift+' : ''}${key}}`
-  }
-
-  _getSpecialKey = (key) => {
-    switch (key) {
-      case '{':
-        return '{'
-      case 'ArrowDown':
-        return 'downarrow'
-      case 'ArrowLeft':
-        return 'leftarrow'
-      case 'ArrowRight':
-        return 'rightarrow'
-      case 'ArrowUp':
-        return 'uparrow'
-      case 'Backspace':
-        return 'backspace'
-      case 'Delete':
-        return 'del'
-      case 'Enter':
-        return 'enter'
-      case 'Escape':
-        return 'esc'
-      case 'Insert':
-        return 'insert'
-      case 'PageDown':
-        return 'pagedown'
-      case 'PageUp':
-        return 'pageup'
-      default:
-        return null
-    }
-  }
-
-  _getKeyValue = (event) => {
-    const { key, altKey, ctrlKey, metaKey } = event
-
-    if (key.length === 1 && key !== '{') {
-      // we explicitly check here so we don't accidentally add shift
-      // as a modifier key if its not needed
-      if (!altKey && !ctrlKey && !metaKey) {
-        return key
-      }
-
-      return this._addModifierKeys(key.toLowerCase(), event)
-    }
-
-    const specialKey = this._getSpecialKey(key)
-
-    if (specialKey) {
-      return this._addModifierKeys(specialKey, event)
-    }
-
-    return ''
-  }
-
   _getMessage = (event, $el) => {
     if (!eventsWithValue.includes(event.type)) {
       return null
     }
 
-    if (event.type === 'keydown') {
-      return this._getKeyValue(event)
+    let val = $el.val()
+
+    if (event.type === 'keydown' || event.type === 'keyup') {
+      val = val.replace(/{/g, '{{}')
+
+      if (event.key === 'Enter') {
+        val = `${val}{enter}`
+      }
     }
 
-    return $el.val()
+    return val
   }
 
   _shouldRecordEvent = (event, $el) => {
     const tagName = $el.prop('tagName')
 
-    return !((tagName !== 'INPUT' && event.type === 'keydown') ||
+    return !((tagName !== 'INPUT' && (event.type === 'keydown' || event.type === 'keyup')) ||
+      (event.type === 'keydown' && event.key !== 'Enter') ||
+      (event.type === 'keyup' && event.key === 'Enter') ||
```

Comment:
Might want to start adding some comments or more readable bool checks for this list of reasons to record, because this is getting harder to read and I'm already starting to forget why some of these are ignored.